plugins/modules/zos_blockinfile.py

```diff
@@ -96,13 +96,15 @@
   marker_begin:
     description:
     - This will be inserted at C({mark}) in the opening ansible block marker.
+    - Value needs to be different from marker_end.
```

Comment:
I think this was mentioned by Ketan before, but wording is different between `marker_begin` and `marker_end`. The later says "Value must be different from marker_end" It would be nice to have the same wording.

Reply:
Thanks for pointing this out, I fixed it and actually another doc issue.